src/ansys/tools/visualization_interface/backends/pyvista/pyvista_interface.py

```diff
@@ -242,7 +242,7 @@ def plot(
                 return self._object_to_actors_map
 
         # Check what kind of object we are dealing with
-        if isinstance(plottable_object, pv.PolyData):
+        if isinstance(plottable_object, pv.PolyData) or isinstance(plottable_object, pv.UnstructuredGrid):
```

Comment:
```suggestion
        if isinstance(plottable_object, (pv.PolyData, pv.UnstructuredGrid)):
```
You can pass a tuple and this will check that it is any of those types